flashinfer/sampling.py

```
@@ -1542,8 +1554,25 @@ def top_p_renorm_probs(
     sampling_from_probs
     top_k_renorm_probs
     """
+    batch_size = probs.size(0)
+    vocab_size = probs.size(1)
+    # Workspace size for AIR Top-P radix algorithm.
+    # Must match GetAirTopPRenormWorkspaceSize in air_top_p.cuh.
+    align256 = lambda x: ((x + 255) // 256) * 256
```

Ruff (0.15.5)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `align256` as a `def`

(E731)